solr/solr-ref-guide/modules/getting-started/pages/introduction.adoc

```diff
@@ -16,28 +16,32 @@
-Any platform capable of HTTP can talk to Solr.
-See xref:deployment-guide:client-apis.adoc[] for details on client APIs.
+Solr's xref:query-guide:query-syntax-and-parsers.adoc[query syntax and parsers] offer support for everything from the simplest keyword searching through to complex queries on multiple fields and xref:query-guide:faceting.adoc[faceted] search results.
+xref:query-guide:collapse-and-expand-results.adoc[Collapsing] and xref:query-guide:result-clustering.adoc[clustering] results offer compelling features for e-commerce and storefronts.
+xref:query-guide:streaming-expressions.adoc[Streaming expressions] allow you to conduct analytics on an entire corpus, a subset matching a query, or a random sample from a set of documents.
+Powerful xref:query-guide:math-expressions.adoc[math expressions] build on streaming expressions to provide the backbone for advanced analysis and predictive analytics use cases.
 
-Flexible schema configurations allow nearly any type of data to be stored in Solr.
-The xref:indexing-guide:schema-elements.adoc[] has more details on these options.
+Advanced relevancy tuning is also supported;
+Solr provides access to almost all of Lucene's text analysis features including tokenization, stemming, synonyms and much more, allowing you to tune relevancy based on knowledge of your users and your domain.
+Solr even allows for customization of relevancy via machine learning using the xref:query-guide:learning-to-rank.adoc[] feature.
 
-Solr offers support for the simplest keyword searching through to complex queries on multiple fields and faceted search results.
-Collapsing and clustering results offer compelling features for e-commerce and storefronts.
-Powerful math expressions provide the backbone for advanced analytics use cases.
-The xref:query-guide:query-syntax-and-parsers.adoc[] has more information about searching and queries.
+Queries are transmitted to Solr via HTTP 1.1 or 2.0 requests and the response is typically a list of structured document descriptors.
+In the classic example, 10 descriptors are returned, each including a URL to locate the document (often rendered as "10 blue links"). However, Solr can go far beyond document locators and many other types of document metadata might also be included.  Flexible schema configurations allow nearly any type of metadata to be associated with a document indexed in Solr.
+The xref:indexing-guide:schema-elements.adoc[schema elements] page of the indexing guide has more details on these options.
 
-If Solr's capabilities are not impressive enough, its ability to handle very high-volume applications should do the trick.
+JSON is the default response format, but it could also be XML, CSV, optimized binary, or (with customization) any format you desire.
+This means that a wide variety of clients will be able to use Solr. Such clients might be web applications, browsers, rich client applications, or mobile devices.
+Any platform capable of HTTP can talk to Solr.
```

Comment:
Should we rephrase to "Any platform capable of communicating in HTTP can be used to interact with Solr."?